### ISSUE TITLE
Adding a static route breaks graphql route

### ISSUE THREAD

**Issue by davidblurton** [kennethreitz/responder#141]
```
api = responder.API()
api.add_route("/graphql", schema)
```

If I add a route for my graphql server, I can post to it and get an api response (from graphiql for example)

```
api = responder.API()
api.add_route("/graphql", schema)
api.add_route("/", static=True)
```

If I add a static route (or any route with `default=True`, I now get the index.html file back. The graphiql interface still loads but when I make a query I get index.html back as the response.

**Comment by taoufik07** [comment]
Working fine for me

**Comment by davidblurton** [comment]
I'm using a custom static dir 
```
api = responder.API(static_dir='build')
```

Perhaps this doesn't do what I thought it did. I thought it would mean it just used the folder build instead of the folder static to serve static assets. 

The problem goes away if I remove static_dir

**Comment by taoufik07** [comment]
It's doing what it was build to do.

* While debugging I've found this bug
graphiql template is loaded into `resp.content`, so it's served first and the static response (loaded into resp.text) is shadowed.

**Comment by davidblurton** [comment]
This makes sense - if the index.html is missing nothing is written into resp.text and the response is still valid json.

**Comment by taoufik07** [comment]
Once graphiql is loaded, it makes some requests, that's when `resp.text` is sent

**Comment by taoufik07** [comment]
Because we sent first `resp.content` if there's any else `resp.text` if it exists than the formats and the media

**Comment by taoufik07** [comment]
Just made a PR #142 please check if it resolves the problem

**Comment by taoufik07** [comment]
Nope, but if you want that behavior don't set static to True

**Comment by davidblurton** [comment]
From the docs:

default – If True, all unknown requests will route to this view.
static – If True, and no endpoint was passed, render “static/index.html”, and it will become a default route.

Static is the default route, and default routes are supposed to match unknown requests. If I've already registered the route, then it's not an unknown request.

**Comment by taoufik07** [comment]
My bad, I get you know !

**Comment by taoufik07** [comment]
Hey @davidblurton, test PR #147

**Comment by davidblurton** [comment]
@taoufik07 Tested it out - this is perfect 🤩

**Comment by taoufik07** [comment]
@kennethreitz guess it's fixed, you can now merge #147 and close this issue :D
